tests/data/included/jazz/tests/functional/TestGuy.php

```diff
@@ -1,10 +1,13 @@
-<?php //[STAMP] 481f1bf8d02786cb5c4fc661670aab66
+<?php
+
+//[STAMP] 481f1bf8d02786cb5c4fc661670aab66
```

Comment:
This class was automatically generated, I will delete it. (all other TestGuys in tests/data/included too)